llvm/test/Assembler/repo-prunning.ll

```diff
@@ -15,24 +15,19 @@
 
```

Comment:
I've missed this before: the file name mis-spells "pruning" as "prunning". Same comment applies to several of these files.

Reply:
Thanks, this issue is fixed in a separate commit (559c5dc1ed55979464fda41a).